Assets/Placenote/LibPlacenote/LibPlacenote.cs

```diff
@@ -405,16 +405,17 @@ static void OnPose (ref PNTransformUnity outputPose, ref PNTransformUnity arkitP
 	{
 		Matrix4x4 outputPoseMat = PNUtility.MatrixOps.PNPose2Matrix4x4 (outputPose);
 		Matrix4x4 arkitPoseMat = PNUtility.MatrixOps.PNPose2Matrix4x4 (arkitPose);
+		MappingStatus status = (MappingStatus)PNGetStatus ();
 
 		var listeners = Instance.listeners;
 		MainThreadTaskQueue.InvokeOnMainThread (() => {
-			foreach (var listener in listeners) {
-				listener.OnPose (outputPoseMat, arkitPoseMat);
+			if (status == MappingStatus.RUNNING) {
```

Comment:
If you move this check up a couple lines you can skip copying the listeners/adding to the queue.